lib/trento/application/projectors/sap_system_projector.ex

```diff
@@ -341,11 +341,11 @@ defmodule Trento.SapSystemProjector do
     sap_system =
       SapSystemReadModel
       |> Repo.get!(sap_system_id)
-      |> Repo.preload([:tags])
+      |> Repo.preload([:tags, :database_instances, :application_instances])
```

Comment:
Something I noticed here as that we could even do this:
```
@impl true
  def after_update(
        %SapSystemRestored{},
        _,
        %{sap_system: %SapSystemReadModel{} = sap_system}
      ) do
    enriched_sap_system =
      Repo.preload(sap_system, [
        :application_instances,
        :database_instances,
        :tags
      ])

    ....
```
I mean, we don't need to query, as the read model already comes in the 3rd parameter of the `after_update`. 
Not a big deal though.